Library/Formula/logstash.rb

```diff
@@ -28,7 +40,7 @@ def caveats; <<-EOS.undent
       output { stdout { codec => rubydebug } }
     EOS
 
-    output = pipe_output("/bin/echo 'hello world' | #{bin}/logstash agent -f simple.conf")
+    output = pipe_output("/bin/echo 'hello world' | #{bin}/logstash -f simple.conf")
```

Comment:
Thanks @dunn, that's better than manually piping. The input also needs a newline lest logstash won't consume it; I've pushed 611611d08bb505d472127124f3268cc54c45b22e incorporating your suggestion.